packages/table/preview/index.tsx

```diff
@@ -97,10 +97,11 @@ const COLUMN_COUNTS = [
 const ROW_COUNTS = [
     0,
     1,
-    10,
+    5,
+    20,
     100,
-    10000,
-    1000000,
+    1000,
+    100000,
```

Comment:
Rendering `1,000,000` cells now throws `Maximum Call Stack Exceeded` errors because I'm saving the rows in memory to retain their order, which I wasn't doing before. So I got rid of that option for now.